--- frc2022/roborio/arm-frc2022-linux-gnueabi/usr/lib/libc.so	2020-10-28 21:59:18.000000000 -0700
+++ frc2022/roborio/arm-frc2022-linux-gnueabi/usr/lib/libc.so	2021-11-27 15:59:22.000000000 -0800
@@ -2,4 +2,4 @@
    Use the shared library, but some functions are only in
    the static library, so try that secondarily.  */
 OUTPUT_FORMAT(elf32-littlearm)
-GROUP ( /lib/libc.so.6 /usr/lib/libc_nonshared.a  AS_NEEDED ( /lib/ld-linux.so.3 ) )
+GROUP ( ../lib/libc.so.6 ../../usr/lib/libc_nonshared.a  AS_NEEDED ( ../lib/ld-linux.so.3 ) )
